app/main/helpers/s3.py

```diff
@@ -21,7 +22,13 @@ def save(self, path, name, file, acl='public-read'):
                 self.bucket_name,
                 full_path
             )
-
         key = self.bucket.new_key(full_path)
-        key.set_contents_from_file(file)
+        key.set_contents_from_file(file,
+                                   headers={'Content-Type':
+                                            self.__get_mimetype(key.name)})
```

Comment:
Should be `_get_mimetype`. Double-underscore methods get name-mangled and are useful only for things you don't want to be overridden by subclasses.